payroll-processor-functions/src/payroll-processor-functions/Features/Employee/EmployeesGetTrigger.cs

```diff
@@ -10,9 +10,9 @@ namespace PayrollProcessor.Functions.Features.Employee
 {
     public class EmployeesGetTrigger
     {
-        [FunctionName("EmployeesGetTrigger")]
+        [FunctionName(nameof(EmployeesGetTrigger))]
```

Comment:
I like this idea - I think in my PR I should use `nameof()` and pass the function method name (in this case it would be `nameof(Run)`)